threat-model/readme.md

```diff
@@ -200,18 +200,25 @@ Helper parties may run either on physical machines owned by directly by the aggr
 
 ### 1.9 Operators of TEEs
 
-As a piece of hardware, TEEs will have an operator with access to the machine. Most commonly, this will be a cloud provider. Depending on the specific hardware, there may be known vulnerabilities in which an attacker who only controls the operator can violate the obliviousness of client/user data. These attacks are outside this threat model, but are likely to inform specific web platform decisions about which instantiations of private computation to support.
+Trusted Execution Environments (TEEs) offer hardware supported confidential computing.
+The operator of a TEE is the entity who controls the physical host machine on which this hardware is installed.
+Most commonly, the operator will be a cloud provider. 
 
 #### 1.9.1 Assets
-TODO
+1. The host machine with the TEE hardware installed.
 
 
 #### 1.9.2 Capabilities
-TODO
+1. The ability to run arbitrary processes on the host machine. TEEs are designed to provide confidentiality and integrity despite this, but, in practice may be vulnerable to side channel leakage via, e.g., the cache or memory access patterns or speculative/transient execution attacks like Spectre. (see Section 1.10 for more on the capabilities of a TEE.)
```

Comment:
Section numbering is a curse, especially when it is manually maintained like this.  Maybe use `[below](#tee-manufacturers)` and add something for that to point to (see below).

```suggestion
1. The ability to run arbitrary processes on the host machine. TEEs are designed to provide confidentiality and integrity despite this, but, in practice may be vulnerable to side channel leakage via, e.g., the cache or memory access patterns or speculative/transient execution attacks like Spectre. (see [below](#tee-manufacturers) for more on the capabilities of a TEE.)
```